corehq/apps/locations/views.py

```diff
@@ -349,6 +349,26 @@ class EditLocationView(NewLocationView):
     page_title = ugettext_noop("Edit Location")
 
     @property
+    def parent_pages(self):
+        breadcrumbs = [{
+            'title': LocationsListView.page_title,
+            'url': reverse(
+                LocationsListView.urlname,
+                args=[self.domain],
+                params={"selected": self.location._id}
+            )
+        }]
+        if self.location.parent_id:
+            breadcrumbs.append({
+                'title': self.location.parent.name,
+                'url': reverse(
+                    EditLocationView.urlname,
+                    args=[self.domain, self.location.parent_id],
+                )
+            })
+        return breadcrumbs
```

Comment:
I thought about it, decided this was the simplest, but if there's interest, I'd be up for adding in the rest of the parents.
As-is this just shows a link to all locations and a link to the immediate parent.